### DISCUSSION TITLE
Missing Armor Addon World Model File

### DISCUSSION THREAD

**Discussion by Noggog** [Mutagen-Modding/Mutagen.Bethesda.Analyzers#83] [Topic Codes]
# Context
Bethesda Path:
`ARMA/[MOD2/MOD3]`
Mutagen Path: 
`ArmorAddon.WorldModel.[M/F].File`

# Description
Has a model listed, but the asset cannot be found in the load order

# Effects
Object is invisible

**Comment by erri120** [comment]
![2021 09 25-12 36 14  SkyrimSE_Q7GUuwyLOB](https://user-images.githubusercontent.com/16577545/134859260-7e37b653-a611-4dad-b394-d8c3e336ac7f.jpg)

**Comment by Elscrux** [comment]
See this: https://ck.uesp.net/wiki/ArmorAddon
It is only an issue with both or only the female armor addon being assigned.
